pkg/command/discovery_source.go

```diff
@@ -143,23 +164,18 @@ func newUpdateDiscoverySourceCmd() *cobra.Command {
 		Use:   "update [name]",
 		Short: "Update a discovery source configuration",
 		Args:  cobra.ExactArgs(1),
-		Example: `
-    # Update a local discovery source. If URI is relative path, 
-    # $HOME/.config/tanzu-plugins will be considered base path
-    tanzu plugin source update standalone-local --type local --uri new/path/to/local/discovery
-
-    # Update an OCI discovery source. URI should be an OCI image.
-    tanzu plugin source update standalone-oci --type oci --uri projects.registry.vmware.com/tkg/tanzu-plugins/standalone:v1.0`,
-
 		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
**Possible future enhancement:** Check whether the provided oci image is a valid plugin-inventory image or not if not throw an error without updating the discovery source.

Reply:
I like this idea.  It would provide a better UX.
However, after thinking about it some more, there is a scenario this would prevent:
- setup the CLI pointing to an air-gapped registry **before** the air-gapped image is ready

Do you think this is a scenario that an air-gapped operator may use?
If so, we could just print a warning that the OCI image configured is not valid, and the operator could tell their users to ignore this.

Reply:
Ok, that make sense.  I'll work on this in a follow-up as it will probably require new functions to allow doing this from the command package.